client/src/components/Form/Elements/FormData/FormData.vue

```diff
@@ -427,7 +428,10 @@ function handleIncoming(incoming: Record<string, unknown> | Record<string, unkno
                     if (config.multiple) {
                         const newValues = currentValue.value ? currentValue.value.slice() : [];
                         incomingValues.forEach((v) => {
-                            newValues.push(v);
+                            // Avoid duplicates
```

Comment:
It seems like we'd ideally want to prevent the drop the same way we prevent dropping an invalid datatype?

Reply:
Ahh good point! I'll check how it is done and update this.

Reply:
I've updated the screencast, now it highlights the warning when the entry is duplicated.

![image](https://github.com/user-attachments/assets/d0d3d917-46b7-4b4d-85e3-077006898c87)